testing/python/transform/test_tilelang_transform_producer_consumer_ws.py

```
@@ -145,6 +145,37 @@ def main(
 
 
+def guarded_prelude_tma_postloop_scalar(block=64, iters=2, dtype="float16", threads=128):
+    """Nested guarded pipeline whose post-loop consumer uses a prelude scalar."""
+
+    @T.prim_func
+    def main(
+        Q: T.Buffer((iters * block * 2, block), dtype),
+        K: T.Buffer((iters * block * 2, block), dtype),
+        LSE: T.Buffer((block * 2, block), dtype),
+        idx: T.Buffer((1,), "int32"),
+        O: T.Buffer((block, block), dtype),
```

Ruff (0.15.13)

[error] Ambiguous variable name: `O`

(E741)